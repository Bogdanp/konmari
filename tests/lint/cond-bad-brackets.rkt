#lang racket/base

(cond
  (void)
  [(void)]
  (#t 'a)
  [#f 'b]
  (0 'c)
  {1 'd}
  [else 'e])
