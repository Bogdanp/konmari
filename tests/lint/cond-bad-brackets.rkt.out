/cond-bad-brackets.rkt:4:3:warning:this cond clause should be surrounded by square brackets
/cond-bad-brackets.rkt:6:3:warning:this cond clause should be surrounded by square brackets
/cond-bad-brackets.rkt:8:3:warning:this cond clause should be surrounded by square brackets
/cond-bad-brackets.rkt:9:3:warning:this cond clause should be surrounded by square brackets
